test/compilable/extra-files/makedeps2.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+grep  "make\\ deps:"  ${RESULTS_DIR}/compilable/makedeps2.dep || exit 1
+grep  'makedeps2.d'  ${RESULTS_DIR}/compilable/makedeps2.dep || exit 1
+grep  'makedeps.sh'  ${RESULTS_DIR}/compilable/makedeps2.dep || exit 1
+grep  'makedeps_a.d'  ${RESULTS_DIR}/compilable/makedeps2.dep || exit 1
+grep  'object.d'  ${RESULTS_DIR}/compilable/makedeps2.dep || exit 1
```

Comment:
Better use `set -e` instead of the constant `|| exit 1`. 
See also: https://github.com/dlang/dmd/pull/8083